### PR TITLE
Adds introspect command

### PR DESCRIPTION
**Pull request by kamilkisiela**
```bash
hive introspect https://example.com/graphql
hive introspect https://example.com/graphql --header "Key:Value"
hive introspect https://example.com/graphql --write schema.graphql
hive introspect "./src/**/*.graphql"
```

### CODE REVIEW

File: packages/libraries/cli/src/commands/introspect.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+import { Args, Flags } from '@oclif/core';
+import Command from '../base-command';
+import { loadSchema } from '../helpers/schema';
+
+export default class Introspect extends Command {
+  static description = 'introspects a GraphQL Schema';
+  static flags = {
+    write: Flags.string({
+      aliases: ['W'],
+      description: 'Write to a file',
```
Comment (n1ru4l) [inline packages/libraries/cli/src/commands/introspect.ts:13]: This could also list the possible file extensions

**Review by n1ru4l — APPROVED** [review]
Approved with optional suggestion

File: .changeset/spotty-panthers-accept.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@graphql-hive/cli': minor
+---
+
+Adds introspect command
```
Comment (n1ru4l) [inline .changeset/spotty-panthers-accept.md:5]: This could include an example usage 😇

Comment (kamilkisiela) [reply .changeset/spotty-panthers-accept.md:5]: it's in readme!

Comment (n1ru4l) [reply .changeset/spotty-panthers-accept.md:5]: Then link to the readme from here 😇
My point is that the changelog is a great place to give an quick example, without people having to dig docs. As I said it is optional.

**Pull request merged**